### PR TITLE
Update ADOPTERS.md

### PR DESCRIPTION
**Pull request by edyoung**
# PR Summary

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Comment by SteveL-MSFT** [comment]
@edyoung You'll need to fix the Codacy issue which is to indent you bullet by 2 spaces.  Also, @TravisEz13 we should consider updating .spelling to ignore this specific file since we're askinging the adopter to submit the PR and we'll likely hit many new words.

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]
Address Codacy issue

File: ADOPTERS.md [hunk 1]
```diff
@@ -11,3 +11,5 @@ Example entry:
 -->
 
 This is a list of adopters of using PowerShell in production or in their products (in alphabetical order):
+
+* [Azure Cloud Shell](https://shell.azure.com/) provides a batteries-included browser-based PowerShell environment used by Azure administrators to manage their environment. It includes up-to-date PowerShell modules for Azure, AzureAD, Exchange, Teams, and many more.
```
Comment (SteveL-MSFT) [inline ADOPTERS.md]:
```suggestion
  * [Azure Cloud Shell](https://shell.azure.com/) provides a batteries-included browser-based PowerShell environment used by Azure administrators to manage their environment.
  It includes up-to-date PowerShell modules for Azure, AzureAD, Exchange, Teams, and many more.
```

File: ADOPTERS.md [hunk 2]
```diff
@@ -11,3 +11,6 @@ Example entry:
 -->
 
 This is a list of adopters of using PowerShell in production or in their products (in alphabetical order):
+
+  * [Azure Cloud Shell](https://shell.azure.com/) provides a batteries-included browser-based PowerShell environment used by Azure administrators to manage their environment.
```
Comment (SteveL-MSFT) [inline ADOPTERS.md]: Looks like I read the Codacy issue wrong, the bullet is left aligned, the content underneath is whitespaced
```suggestion
* [Azure Cloud Shell](https://shell.azure.com/) provides a batteries-included browser-based PowerShell environment used by Azure administrators to manage their environment.
```

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Review by SteveL-MSFT — APPROVED** [review]
LGTM, the Codacy markdown rule seems incorrect.  It's complaining about the spacing between the `*` and the content.

We can debate the .spelling separate from this PR

File: ADOPTERS.md [hunk 3]
```diff
@@ -11,3 +11,6 @@ Example entry:
 -->
 
 This is a list of adopters of using PowerShell in production or in their products (in alphabetical order):
+
+* [Azure Cloud Shell](https://shell.azure.com/) provides a batteries-included browser-based PowerShell environment used by Azure administrators to manage their environment.
+  It includes up-to-date PowerShell modules for Azure, AzureAD, Exchange, Teams, and many more.
```
Comment (TravisEz13) [inline ADOPTERS.md]:
```suggestion
  It includes up-to-date PowerShell modules for Azure, AzureAD, Exchange, Teams, and many more.
  More information about Azure Cloud Shell is available at [Azure Cloud Shell Overview.](https://docs.microsoft.com/en-us/azure/cloud-shell/overview)
```

File: ADOPTERS.md [hunk 4]
```diff
@@ -11,3 +11,7 @@ Example entry:
...
+  More information about Azure Cloud Shell is available at [Azure Cloud Shell Overview.](https://docs.microsoft.com/en-us/azure/cloud-shell/overview)
```
Comment (TravisEz13) [inline ADOPTERS.md]:
```suggestion
  More information about Azure Cloud Shell is available at [Azure Cloud Shell Overview.](https://docs.microsoft.com/azure/cloud-shell/overview)
```

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 1 hour

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 hour ago

File: ADOPTERS.md [hunk 5]
```diff
@@ -11,3 +11,7 @@ Example entry:
 -->
 
 This is a list of adopters of using PowerShell in production or in their products (in alphabetical order):
+
+* [Azure Cloud Shell](https://shell.azure.com/) provides a batteries-included browser-based PowerShell environment used by Azure administrators to manage their environment.
+  It includes up-to-date PowerShell modules for Azure, AzureAD, Exchange, Teams, and many more.
```
Comment (TravisEz13) [inline ADOPTERS.md]:
```suggestion
  It includes up-to-date PowerShell modules for `Azure`, `AzureAD`, `Exchange`, `Teams`, and many more.
```

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 1 hour

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 hour ago

**Comment by iSazonov** [comment]
It would be nice to have an soft integrations with adopters CIs so that we could see their daily status for latest pwsh builds - this helps us to discover, track and resolve breaking changes for tools (now we have only formal check boxes in PR template).

**Comment by TravisEz13** [comment]
@iSazonov Please file a new issues for new enhancements rather than commenting in an existing PR/Issue

**Pull request merged**